src/reducers/index.js

```diff
@@ -1,6 +1,10 @@
 import { combineReducers } from 'redux';
+import { reducer as formReducer } from 'redux-form';
 import placeholderReducer from './placeholder';
+import authReducer from './auth';
 
 export default combineReducers({
   placeholder: placeholderReducer,
+  auth: authReducer,
```

Comment:
I don't think the placeholder reducer needs to be here. Please remove and delete the file also.